server/events/yaml/valid/global_cfg.go

```diff
@@ -180,37 +187,66 @@ func (g GlobalCfg) MergeProjectCfg(log logging.SimpleLogging, repoID string, pro
 				}
 				log.Debug("overriding server-defined %s with repo-specified workflow: %q", WorkflowKey, workflow.Name)
 			}
+		case DeleteSourceBranchOnMergeKey:
+			log.Debug("global deleteSourceBranchOnMerge: %t", deleteSourceBranchOnMerge)
```

Comment:
There needs to be a tradeoff between the amount of info we get during debug mode, vs. readability. This is a bit too much logging lol.  Can we condense this?

Reply:
I eleminated the unnecessary comments